### DISCUSSION TITLE
Multi-platform

### DISCUSSION THREAD

**Discussion by trymeouteh** [quillpad/quillpad#55] [Q&A] [answered]
How would you create the app to run on Windows, Mac, Linux and iOS? Will you be using flutter?

**Comment by jsixface** [comment, marked as answer]
Once the app is moved to Jetpack Compose (at least the main editor view), we can use Compose for Desktop https://www.jetbrains.com/lp/compose-desktop/
It uses the view APIs that are the same as android Jetpack Compose.
Arumugam Jeganathan (Aru)

**Reply by jsixface** [reply]
Updated the readme: https://github.com/quillpad/quillpad?tab=readme-ov-file#milestone-16-jetpack-compose

This is not going to be done.
